README.md

```diff
@@ -1 +1,4 @@
 # yadl
+
+Documentation : https://github.com/YADL/yadl/wiki
```

Comment:
I do not think this will render correctly in Markdown.  You may need to change it like this:

```
Documentation is available in the [Wiki](https://github.com/YADL/yadl/wiki)
```